src/bootstrap/bin/rustc.rs

```diff
@@ -126,6 +130,13 @@ fn main() {
         }
     }
 
+    if is_proc_macro && stage == "0" {
+        // Cargo has a bug where it doesn't pass RUSTFLAGS to proc_macros:
```

Comment:
I.. wouldn't call this a bug :) More like 'unexpected behavior'. https://github.com/rust-lang/cargo/issues/4423 seems like the more appropriate issue to cite, though.

Reply:
Oops, yes, that's the one I meant to link to. The issue title isn't quite right though - we're neither cross-compiling nor building a build script.